### PR TITLE
add portal clear function

### PR DESCRIPTION
**Pull request by pwz1985**
First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [ ] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [ ] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [ ] Rebase before creating a PR to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [ ] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

### CODE REVIEW

File: components/portal/portal-manager.tsx [hunk 1]
```diff
@@ -37,6 +37,10 @@ export default class PortalManager extends React.PureComponent<
     this.setState(state => ({
       portals: state.portals.filter(item => item.key !== key),
     }));
+  clear = () =>
+    this.setState({
```
Comment (bang88) [inline components/portal/portal-manager.tsx]: 如果就这样清理的话会把`Modal` 以及所有Portal都清理掉。 估计不太妥？

Comment (pwz1985) [reply components/portal/portal-manager.tsx]: 可以保留这个方法，由开发者自己取舍，很多简单业务场景，使用那个 remove很繁琐

**Comment by bang88** [comment]
首选 谢谢PR。

但是这个不建议这么做，`clear`的本意也是只 清除`Toast`，没有其他意思，如果清除了不该清除的东西反而不好，让开发者自己斟酌不是很好，一般都会偷懒全都清理，用户往往不想看到这中情况。而且很多时候是不需要自己去清除。

File: components/portal/portal-manager.tsx (hunk 1 above)
Comment (bang88) [reply components/portal/portal-manager.tsx]: 暂时不能，因为key都是number 没有加前缀，所以没办法只清除Toast。

**Pull request closed without merging**